src/main/java/nextstep/line/entity/Line.java

```diff
@@ -0,0 +1,79 @@
+package nextstep.line.entity;
+
+import nextstep.section.entity.Sections;
+
+import javax.persistence.*;
+
+@Entity
+public class Line {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "LINE_ID")
+    private Long id;
+
+    @Column(length = 20, nullable = false)
+    private String name;
+
+    @Column(length = 20, nullable = false)
+    private String color;
+
+    @Column(nullable = false)
+    private Long distance;
+
+    @Embedded
+    private Sections sections;
+
+    protected Line() {
+    }
+
+    public Line(String name, String color, Long distance, Sections sections) {
+        this.name = name;
+        this.color = color;
+        this.distance = distance;
+        this.sections = sections;
```

Comment:
주 생성자를 호출하는 방식으로 변경해보면 어떨까요?
```suggestion
        this(null, name, color, distance, sections);
```